hooks/useUser.tsx

```diff
@@ -19,12 +19,19 @@ const useUser = () => {
 
   useEffect(() => {
     const init = async () => {
+      const req = JSON.stringify({ email });
+      const headers = {
+        "Content-Type": "application/json", // Specify the content type
+      };
+      fetch("/api/email", {
```

Comment:
The fetch call for /api/email does not handle the response, which could lead to silent errors. Also, having two fetch calls (to /api/email and /api/account) can be optimized if needed.